docs/reference/integrations.rst

```diff
@@ -108,6 +109,17 @@ You can override this behavior to use a native pydantic validation/serialization
 
 .. literalinclude:: /examples/reference/integrations/native_pydantic.py
 
+.. _msgspec:
+
+Msgspec
```

Comment:
Msgspec looks weird to me. They do not use upper letter in the official doc. I think it would be better to name section as "Working with msgspec"